docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,146 @@
+
+![concrete-products](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-4-concrete-products.png)
+
+![create-offer](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-4-create-offer.png)
+
+
+### 5) Operator: Set up the payment process
+
+The following steps are taken by the operator:
+1. Decide what kind of payment process to implement—for example, whether to use a PSP or not. 
+2. Choose a revenue model to apply in the marketplace—for example, transaction-based, subscription-based, or listing-based—and whether a PSP can cover it.
+
+The following example illustrates the process when using a PSP to cover the marketplace payments completely:
+1. The merchant registers on the PSP portal and provides relevant company and representative information.
+2. The PSP checks the merchant during the compliance and KYC process.
+3. PSP approves the merchant and starts technical onboarding.
+4. The PSP calculates transactional fees and commissions agreed upon and makes payments to merchants.
+5. The merchant receives access to the PSP Back Office, where data regarding commissions and payments can be checked.
+
+{% info_block infoBox %}
+
+For more information about the payment setup and available options, ask your Spryker contact.
+
+{% endinfo_block %}
+
+### 6) Operator: Set up the order management process
+
+The State Machine models the order management process, including both delivery and return parts that can be defined per merchant. It helps if you have big and small merchants with different fulfillment processes. Otherwise, you can use one State Machine flow for all your merchants.
+
+
+There are two fulfillment models applicable to Marketplace you can choose:
+* Fulfillment by merchants: Merchants receive, process, and dispatch orders directly to your marketplace buyers.
+* Fulfillment by marketplace: Merchants sell products and the operator manages the logistics, including returns.
+
+The shipping process needs to be set up on a project level.
+
+The State Machine can be set up to route an automated email to the relevant merchant once an order is created.
+
+The following diagram shows an example of a simple State Machine workflow, where every merchant handles their own shipments and returns.
+
+![state-machine-workflow-example](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-6-state-machine-workflow-example.png)
+
+### 7) Merchant and operator: Approve products and offers and activate the merchant
+
+The following steps are taken by the operator:
+
+1. Do a final check of the merchant's public profile, products, and offer quality. 
+2. Activate merchants and approve products and offers in the Back Office. 
```

Comment:
consider adding links to this pages https://docs.spryker.com/docs/pbc/all/merchant-management/202212.0/marketplace/manage-in-the-back-office/manage-merchants.html#activating-and-deactivating-merchants, https://docs.spryker.com/docs/marketplace/user/back-office-user-guides/202212.0/marketplace/offers/managing-merchant-product-offers.html#approving-or-denying-offers. Also, how do you approved products?

Reply:
@mikost7 the second question is for you